internal/rbd/rbd_util.go

```diff
@@ -1631,7 +1634,7 @@ func (ri *rbdImage) getParent() (*rbdImage, error) {
 	if err != nil {
 		return nil, err
 	}
-	if ri.ParentName == "" {
+	if ri.ParentName == "" || ri.ParentInTrash {
```

Comment:
Looks like we are going to get ParentName even it its in trash, pleas add a comment here as well.

Reply:
It does have name.
We just can't refer to it by its name when it is in trash.

Reply:
thats strange, can you please add a comment about it.